code/client/clrcore/External/PropHash.cs

```diff
@@ -0,0 +1,11456 @@
+		ss1_02_building00 = -1087287778,
+		ss1_02_building00_dtl = 1317947300,
+		ss1_02_building00_dtl_1 = 1401152204,
+		ss1_02_building00_dtl_2 = 21773918,
+		ss1_02_building01 = -1426709080,
+		ss1_02_building01_dtl = -724395030,
+		ss1_02_building01_dtl2 = -279744443,
+		ss1_02_building03 = -2058855863,
+		ss1_02_building03_dtl = 1958430556,
+		ss1_02_building03_dtl_1 = -1543537378,
+		ss1_02_decal = -1780886489,
+		ss1_02_decal2 = 104890951,
+		ss1_02_decal3 = -1393569881,
+		ss1_02_decal4 = -1161270440,
+		ss1_02_decal5 = 1333040302,
+		ss1_02_grd01 = -1732998491,
+		ss1_02_hedge_dcl = 1602125812,
+		ss1_02_hedge_dcl2 = -1441045355,
+		ss1_02_pool_water = 1919516466,
+		ss1_02_ss1_emissive_ss1_02 = -500045280,
+		ss1_02_ss1_emissive_ss1_02b = -1920008361,
+		ss1_07_bck_fizz1_lod = -521428309,
+		ss1_07_sgar_rail_lod = 977900690,
+		ss1_10_bld = 210369767,
+		ss1_10_bld_dt01 = -2019265798,
+		ss1_10_bld_trim01 = 1562753294,
+		ss1_10_bld_trim02 = 1257739442,
+		ss1_10_decals01 = 341303678,
+		ss1_10_decals02 = -1799331259,
+		ss1_10_decals03 = 865509363,
+		ss1_10_grd = 123852215,
+		ss1_10_railing01 = 377613226,
+		ss1_10_ss1_emissive_ss1_10 = -1967807737,
+		ss1_12_night_slod = 707438599,
+		ss1_13_grd03_sp_lod = -309550683,
+		ss1_13_night01_slod = 1580084506,
+		ss1_lod_emissive_05 = 557307319,
+		ss1_lod_emissive_slod3 = -1110058314,
+		ss1_lod_slod3 = 2007050028,
+		ss1_props_bug1480687_slod = -1780655002,
+		ss1_props_cablemesh63 = -1640273836,
+		ss1_props_cablemesh63217 = 1652262705,
+		ss1_props_cablemesh6334 = 928902614,
+		ss1_props_cablemesh63621 = 2095548941,
+		ss1_props_cablemesh641 = -1410063030,
+		ss1_props_cablemesh947927 = 1463686267,
+		ss1_props_cablemesh947938 = 43804629,
+		ss1_props_cablemesh947946 = -855441989,
+		ss1_props_cablemesh947955 = 1864583097,
+		ss1_props_cablemesh9479554 = -1853036408,
+		ss1_props_cablemesh9479767 = -1940859617,
+		ss1_props_cablemesh947978 = -1584286919,
+		ss1_props_cablemesh94799 = 78130487,
+		ss1_props_cablemesh947994 = 1182593489,
+		ss1_props_cablemesh9480 = -1593736670,
+		ss1_props_cablemesh94801 = 1806008239,
+		ss1_props_cablemesh94802 = -92136086,
+		ss1_props_cablemesh94803a = -2050051847,
+		ss1_rd1_00 = -1837919433,
+		ss1_rd1_02 = -1393997790,
+		ss1_rd1_03 = -1247225431,
+		ss1_rd1_04 = -1958705959,
+		ss1_rd1_05 = -663347389,
+		ss1_rd1_06 = -961250368,
+		ss1_rd1_07 = -289125409,
+		ss1_rd1_08 = -469092757,
+		ss1_rd1_09 = 169935508,
+		ss1_rd1_10 = 1609977121,
+		ss1_rd1_11 = -152765696,
+		ss1_rd1_12 = -929620379,
+		ss1_rd1_13 = -914284487,
+		ss1_rd1_14 = -1683340148,
+		ss1_rd1_15 = 470598995,
+		ss1_rd1_16 = 769845503,
+		ss1_rd1_17 = 310030895,
+		ss1_rd1_18 = -503197382,
+		ss1_rd1_19 = -1907152414,
+		ss1_rd1_20 = 817229101,
+		ss1_rd1_21 = 520636882,
+		ss1_rd1_22 = 319369684,
+		ss1_rd1_24 = 2041413403,
+		ss1_rd1_25 = 1743903652,
+		ss1_rd1_26 = -546026801,
+		ss1_rd1_27 = -248844740,
+		ss1_rd1_28 = 2077754264,
+		ss1_rd1_29 = -19
```

Comment:
Are you sure those are *prop* model hashes? There's lots of ped/vehicle model hashes.

Reply:
yes as most vehicles can be spawn as object (not interactable)